KellyRyanGlobal/testing-app
# Kelly 5 create jenkins pipeline

To test

1. Pull the branch `kelly-5-create-jenkins-pipeline`
2. Run the instructions in **sample test plan** under 3.3.2 procedure
3. replace step 10 with **kelly-5-create-jenkins-pipeline**
4. Save and build the application
5. Verify the test turns green as the procedure states
